src/orange/BoundingBox.hh

```diff
@@ -122,7 +122,7 @@ BoundingBox<T>::lower() const
  * Upper bbox coordinate (must be valid).
  */
 template<class T>
-CELER_FUNCTION typename BoundingBox<T>::Array3 const&
+CELER_FUNCTION typename BoundingBox<T>::Real3 const&
```

Comment:
Since you're changing these, write the `lower` and `upper` return types as `auto` and use `-> Real3`.

Reply:
You mean:

```
CELER_FUNCTION auto BoundingBox<T>::upper() const -> typename BoundingBox<T>::Real3 const&
````
correct?

Reply:
Just saw this: and I meant `-> Real3 const&` sorry